packages/nextjs/app/builders/0x3566ce434c0d8B80934354c806f8651e1cC91EDD/page.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Image from "next/image";
+
+const umutarray = () => {
```

Comment:
Consider renaming the component to Umutarray to follow React's PascalCase convention.
Also, it's recommended to use the NextPage type from the next package for defining page components in Next.js.